.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    flags:
+      - -trimpath
+    ldflags:
+      - "-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}"
```

Comment:
Is this expecting the default branch to be ```main``` rather than ```master```?

Reply:
Main is referring to the main package in this case. I couldn't find any exact documentation on this (came from scaffolding and see it in other providers), but I'm guessing the terraform sdk pulls those package level variables in for metadata information.